recipes/cpnest/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "cpnest" %}
+{% set version = "0.9.7" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "d698aa9ebf924a6a5e54c29165b75cf67f38616179256495e02e4e75391c718f"
```

Could you please remove the quotes from ``sha256``?
```suggestion
  sha256: d698aa9ebf924a6a5e54c29165b75cf67f38616179256495e02e4e75391c718f
```

---

@marcelotrevisani, I'm happy to do this, but why?

---

It is because the ``conda-forge`` bot was having a bad time to parse the ``sha256`` with ``quotes``. The new release of ``conda-build`` will update this part too.
So, just to avoid any kind of problem it is better to unquote the ``sha256``